README.md

```diff
@@ -1,14 +1,102 @@
 # qorus-vscode
 
-A developer's frontend for Qorus Integration Engine
+A developer's front end for Qorus Integration Engine
 
 https://qoretechnologies.com/qorus_integration_engine/
 
 ## Features
 
-This extension adds support for the following:
+The extension makes possible remote deployment of edited Qorus source files directly from the VSCode editor.
 
-- Remote development
-- Qorus objects templates
-- etc.
+## How to use
 
+### Configuration data and corresponding tree view
+
+In order to work with the extension a project must be open in the VSCode editor. By project we mean the root directory of a project (the root of a git repository, for a typical example).
+
+Next, in the root directory there needs to be a project configuration file named **qorusproject.json** defining development environments and Qorus instances used during the project development. If the configuration file **qorusproject.json** is not there it will be generated from a template so that the user can simply edit its contents seeing the required structure.
```

Comment:
When it will be generated? I hope it should be on demand only, right? Then the command should be mentioned here.

Reply:
I assume that generating the config file only on demand will be released in the next release and README will be updated along with that change.